src/checker/hunspell.rs

```diff
@@ -29,6 +29,33 @@ pub struct HunspellWrapper(pub Arc<Hunspell>);
 unsafe impl Send for HunspellWrapper {}
 unsafe impl Sync for HunspellWrapper {}
 
+// The value is `true` if string is made of emoji's
+// or unicode `VULGAR FRACTION`
+pub fn vulgar_fraction_or_emoji(word: String) -> bool {
+    if word.is_empty() {
+        return false;
+    }
+    for c in word.clone().chars() {
+        if is_vulgar_fraction(c) || unic_emoji_char::is_emoji(c) {
+            continue;
+        } else {
+            return false;
+        }
+    }
+    return true;
+}
+
+// The value is `true` if given character is a vulgar fraction
+pub fn is_vulgar_fraction(character: char) -> bool {
+    let unicode_name = unicode_names2::name(character).map(|n| n.to_string());
+    let vulgar = "VULGAR FRACTION";
```

Comment:
Since this is going to be a rather hot function being called very often, it should really be fast. So a string comparison is a bit hard to justify. If we can check for the character page that would be way more efficient.